tests/api/geom/to_geo.rs

```diff
@@ -39,3 +41,67 @@ fn from_vertex() {
 
     assert_relative_eq!(result, expected, epsilon = 1e-6);
 }
+
+#[test]
+fn to_simple_polygon() {
+    // hexagon cell in Le vigan at low res: https://h3geo.org/#hex=893961acb53ffff
+    let cell = CellIndex::try_from(0x0893_961a_cb53_ffff).expect("index");
+    let polygon = cell_to_multi_polygon(cell);
```

Comment:
probably need to update those test to use `from` and remove/update the `from_cell` test.

Actually, maybe just:
- delete `from_cell`
- rename `to_simple_polygon` to `from_cell`
- rename `to_multi_polygon` to `from_transmeridian_cell`

and we should be good